### ISSUE TITLE
Loading noVNC fails in IE11 with 

### ISSUE THREAD

**Issue by beckjim** [novnc/noVNC#864]
Hello noVNC-team,

I'm trying to use the latest commit of noVNC as of today (Latest commit be70fe0 14 days ago).
IE11 on Win7/Win10 gives the following error when loading noVNC (via vnc.html):

```
noVNC encountered an error:

Unable to get property 'src' of undefined or null reference

https://test.local/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js:1300:3
TypeError: Unable to get property 'src' of undefined or null reference
   at WorkerPool (https://test.local/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js:1300:3)
   at Anonymous function (https://test.local/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js:1368:1)
   at Anonymous function (https://test.local/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js:4:3)
   at Global code (https://test.local/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js:1:2)_
```

Has anybody an idea how to fix this? Unfortunately this Javascript stuff is too heavy for me ...
Thank you !

**Comment by CendioOssman** [comment]
Seems to be fallout from aadcf47d6fccfa4b07f88db000580fe55b1588d5. I'll have a look.

**Comment by CendioOssman** [comment]
Fixed in edb7879927c18dd2aaf3b86c99df69ba4fbb0eab.

**Comment by beckjim** [comment]
Cool, that was quite fast ...
Confirmed. Works like a charm now ...
Thanks a lot!
